azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -101,7 +101,7 @@
   private ExecutingManagerUpdaterThread executingManager;
   // 12 weeks
   private static final long DEFAULT_EXECUTION_LOGS_RETENTION_MS = 3 * 4 * 7
-      * 24 * 60 * 60 * 1000l;
```

Comment:
It seems that your diff base is old since this patch contains a number of changes from #508. Can you please rebase?